clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -5444,6 +5444,43 @@ AST_MATCHER(FunctionDecl, isDefaulted) {
   return Node.isDefaulted();
 }
 
+/// Matches trivial methods and types.
```

Comment:
I do not think is actually matches `types`, but rather `classes` or `CXXRecordDecl`.
And the question is: should it?

I believe we could extend this matcher to also work on types and type locs.
Do you see any downsides to that? (same for the other matcher)